README.md

```diff
@@ -849,33 +849,42 @@ In `src/api/api.cc` we have:
 static Local<ObjectTemplate> ObjectTemplateNew(
     i::Isolate* isolate, v8::Local<FunctionTemplate> constructor,
     bool do_not_cache) {
+  ...
   i::Handle<i::Struct> struct_obj = isolate->factory()->NewStruct(
       i::OBJECT_TEMPLATE_INFO_TYPE, i::AllocationType::kOld);
-  i::Handle<i::ObjectTemplateInfo> obj = i::Handle<i::ObjectTemplateInfo>::cast(struct_obj);
-  InitializeTemplate(obj, Consts::OBJECT_TEMPLATE);
-  int next_serial_number = 0;
-  if (!constructor.IsEmpty())
-    obj->set_constructor(*Utils::OpenHandle(*constructor));
-  obj->set_data(i::Smi::zero());
+  i::Handle<i::ObjectTemplateInfo> obj =
+      i::Handle<i::ObjectTemplateInfo>::cast(struct_obj);
+  {
+    // Disallow GC until all fields of obj have acceptable types.
+    i::DisallowGarbageCollection no_gc;
+    i::ObjectTemplateInfo raw = *obj;
+    InitializeTemplate(raw, Consts::OBJECT_TEMPLATE, do_not_cache);
+    raw.set_data(0);
+    if (!constructor.IsEmpty()) {
+      raw.set_constructor(*Utils::OpenHandle(*constructor));
+    }
+  }
   return Utils::ToLocal(obj);
 }
 ```
 What is a `Struct` in this context?  
 `src/objects/struct.h`
 ```c++
-#include "torque-generated/class-definitions-tq.h"
+#include "torque-generated/src/objects/struct-tq.inc"
```

Comment:
Refer to https://github.com/v8/v8/blob/ee74e718339223345b42935e9f4cbc7be208e557/src/objects/struct.h#L17